### ISSUE TITLE
Auto parse reference to issues/pr in metrics insights

### ISSUE THREAD

**Issue by lowlighter** [lowlighter/metrics#254]
On metrics insights, for each event content like: 
![](https://user-images.githubusercontent.com/22963968/115754931-068c8f00-a39d-11eb-91af-65415eda6161.png)

We should replace issues/prs by `https://github.com/user-repo/issues/xx` (if xx is a pr github is smart enough to redirect to pull)

We need to patch these (there should be 3/4 of them, for comment, issues, pr, and maybe releases event types)
https://github.com/lowlighter/metrics/blob/a0489a42a133818321ee1865fb3460af59ac3ae9/source/app/web/statics/about/index.html#L193

It should call a method which is able to parse these reference (probably just need a regex replace in links):
https://github.com/lowlighter/metrics/blob/a0489a42a133818321ee1865fb3460af59ac3ae9/source/app/web/statics/about/script.js#L51

Below is a reference of reference type I know:
- **issue/pr**:  #254
- **comment**: https://github.com/lowlighter/metrics/pull/237#issuecomment-821993398
- **external issue**: https://github.com/lowlighter/lowlighter/pull/305#issue-620707570

Which currently render as the following:
![](https://user-images.githubusercontent.com/22963968/115758161-1b6a2200-a39f-11eb-9d55-ab4bd65ec99d.png)

Feel free to open a PR 🙂

**Comment by Nixinova** [comment]
Maybe add the following inside the `switch` statement:
```js
case "comment":
 return value
  .replace(/https?:\/\/(?:www\.)?github.com\/(\w+\/\w+)\/\w+\/(\d+)(?:\?\S+)?(#\S+)?/g, (url, repo, id, comment) => `${repo}#${id}` + (comment ? ` (comment)` : '')) // 'lowlighter/metrics#123'
            
```

https://github.com/lowlighter/metrics/blob/a0489a42a133818321ee1865fb3460af59ac3ae9/source/app/web/statics/about/script.js#L52-L57

Does Metrics already parse Markdown? It doesn't parse links at least. It if doesn't then that could be grouped into this.

**Comment by lowlighter** [comment]
Markdown parsing was implemented in #237 with [marked.js](https://github.com/markedjs/marked) ([ref to function](https://github.com/lowlighter/metrics/blob/7d7297fcbe2e329068d2731eb0390808da68d5fc/source/app/metrics/utils.mjs#L163-L196)) but website is still on previous release so it isn't live yet.

The regex seems to match exactly what's needed, so replacing occurrences of `v-html="event.content"` by `v-html="format('comment', event.content)"` should do the trick 🙂 
It's just missing the simple case for implicit local refs (like #254) but that shouldn't be too hard to implement since we can just pass `repo` name as an option and craft the `<a>` tag
